src/Feedback.Analyzer.Backend/Feedback.Analyzer.Infrastructure/FeedbackAnalysisWorkflows/Services/FeedbackAnalysisWorkflowService.cs

```diff
@@ -0,0 +1,78 @@
+using Feedback.Analyzer.Domain.Enums;
+using Feedback.Analyzer.Infrastructure.FeedbackAnalysisWorkflows.Validators;
+using Feedback.Analyzer.Persistence.Extensions;
+using Feedback.Analyzer.Persistence.Repositories.Interfaces;
+using FluentValidation;
+
+namespace Feedback.Analyzer.Infrastructure.FeedbackAnalysisWorkflows.Services;
+
+public class FeedbackAnalysisWorkflowService(
+    IFeedbackAnalysisWorkflowRepository feedbackAnalysisWorkflowRepository,
+     FeedbackAnalysisWorkflowValidator analysisWorkflowValidator
+    ) : IFeedbackAnalysisWorkflowService
+{
+    public IQueryable<FeedbackAnalysisWorkflow> Get(
+        Expression<Func<FeedbackAnalysisWorkflow, bool>>? predicate = default,
+        QueryOptions queryOptions = default)
+        => feedbackAnalysisWorkflowRepository.Get(predicate, queryOptions);
+    
+    public IQueryable<FeedbackAnalysisWorkflow> Get(
+        FeedbackAnalysisWorkflowFilter feedbackAnalysisWorkflowFilter,
+        QueryOptions queryOptions = default)
+        => feedbackAnalysisWorkflowRepository.Get().ApplyPagination(feedbackAnalysisWorkflowFilter);
+    
+    public ValueTask<FeedbackAnalysisWorkflow> GetByIdAsync(
+        Guid feedbackAnalysisWorkflowId,
+        QueryOptions queryOptions = default,
+        CancellationToken cancellationToken = default)
+        => feedbackAnalysisWorkflowRepository.GetByIdAsync(feedbackAnalysisWorkflowId, queryOptions, cancellationToken);
+    
+    public ValueTask<FeedbackAnalysisWorkflow> CreateAsync(
+        FeedbackAnalysisWorkflow feedbackAnalysisWorkflow,
+        CommandOptions commandOptions = default,
+        CancellationToken cancellationToken = default)
+    {
+        var validationResult = analysisWorkflowValidator
+            .ValidateAsync(feedbackAnalysisWorkflow,
+                options =>
+                    options.IncludeRuleSets(EntityEvent.OnCreate.ToString()), cancellation: cancellationToken);
+
+        if (!validationResult.Result.IsValid)
+            throw new ValidationException(validationResult.Result.Errors);
+
+        return feedbackAnalysisWorkflowRepository.CreateAsync(feedbackAnalysisWorkflow, commandOptions,
+            cancellationToken);
+    }
+
+    public async ValueTask<FeedbackAnalysisWorkflow> UpdateAsync(
+        FeedbackAnalysisWorkflow feedbackAnalysisWorkflow,
+        CommandOptions commandOptions = default,
+        CancellationToken cancellationToken = default)
+    {
+        var foundAnalysisWorkflow = await GetByIdAsync(feedbackAnalysisWorkflow.Id, cancellationToken: cancellationToken) ?? throw new InvalidOperationException();
+        
+        foundAnalysisWorkflow.AnalysisWorkflow.Name = feedbackAnalysisWorkflow.AnalysisWorkflow.Name;
+        foundAnalysisWorkflow.AnalysisWorkflow.Type = feedbackAnalysisWorkflow.AnalysisWorkflow.Type;
+       
+        return await feedbackAnalysisWorkflowRepository.UpdateAsync(feedbackAnalysisWorkflow, commandOptions, cancellationToken);
+    }
```

Comment:
Remove update method, currently there's nothing to update in feedback analysis workflow